### PR TITLE
fix: Crash on selecting Centers screen from Nav Drawer

### LINKED ISSUE #457: Crash on Selecting Centers from Navigation Drawer
**Summary:** 

App crashes on selecting Centers from Navigation Drawer. 

**Steps to reproduce:** 

From Navigation Drawer select Centers. 

**Expected behavior:** 

Center List should have appeared.

**Observed behavior:** 

App Crashes.

**Device and Android version:** 

AVD - Marshmallow API 23

**Screenshots:**

### PR DESCRIPTION
**Pull request by tarun0**
Fix #457 
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by mayank-kgp** [comment]
@tarun0  This was happened due to change in a PR where @therajanmaurya  was implementing search interface. I think if he can change there then it won't be necessary.

**Comment by tarun0** [comment]
@mayank-kgp A student is working on the task and the crash was hindering his progress. So, I considered sending the fix so that he may complete it. If Rajan fixes that in his PR, that'll be fine as well. :)

**Comment by mayank-kgp** [comment]
@tarun0  fine :)

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/CentersActivity.java [hunk 1]
```diff
@@ -25,13 +26,20 @@
...
+    private final String LOG_TAG = this.getClass().getSimpleName();
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_toolbar_container);
         showBackButton();
-        centerId = getIntent().getExtras().getInt(Constants.CENTER_ID);
+
+        try {
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/CentersActivity.java]: This is not good idea to put try catch.
Here is Fix 
```java
 @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_toolbar_container);
        showBackButton();
        
        Bundle bundle= getIntent().getExtras();
        if (bundle != null) {
            int centerId = bundle.getInt(Constants.CENTER_ID);
            replaceFragment(GroupListFragment.newInstance(centerId), true, R.id.container);
        } else {
            replaceFragment(new CenterListFragment(), false, R.id.container);
        }
    }
```

and Remove the private int centerId = 0 variable also.

**Comment by tarun0** [comment]
@therajanmaurya Made the changes.

**Pull request merged**